homeassistant/config_entries.py

```diff
@@ -3116,6 +3116,7 @@ def async_update_reload_and_abort(
         options: Mapping[str, Any] | UndefinedType = UNDEFINED,
         reason: str | UndefinedType = UNDEFINED,
         reload_even_if_entry_is_unchanged: bool = True,
+        version: int | UndefinedType = UNDEFINED,
```

Comment:
Let's not update core in this PR, let's stick with your proposal as I did not know it existed

Reply:
Reverted 👍